lib/cppcheck.cpp

```diff
@@ -299,7 +299,11 @@ unsigned int CppCheck::checkFile(const std::string& filename, const std::string
             }
         }
 
-        preprocessor.loadFiles(tokens1, files);
+        try {
+            preprocessor.loadFiles(tokens1, files, true);
```

Comment:
hmm.. an alternative would be:

    if (!preprocessor.loadFiles(tokens1, files))
        return mExitCode;